### PR TITLE
Added portraits

### PR DESCRIPTION
**Pull request by sejtalk**
Changelog:
- Added remade portraits for Genn, Medivh and Nathanos.
- Added remade portraits for Cenarius, Malfurion, Tyrande, Maiev and Illidan.
- Added Jaina portraits.

Secret changelog:
- Added Khadgar portrait that's currently unused.
- Added portraits that were supposed to be added in TFT branch but are added here because the branch is on hold.
- Refactored `modifier` into `trigger` portrait conditions.
- Made `factor` of blank layer 1 while static portraits got 100. This should make it possible to remove conditions of blank layer, since the game prefers layers with 100 factor and instantly turns them on.

### CODE REVIEW

File: common/game_rules/000_wc_game_rules.txt [hunk 1]
```diff
@@ -188,4 +188,18 @@ halfbreed_race = {
 		text = "RULE_HALFBREED_RACE_RANDOM"
 		desc = "RULE_HALFBREED_RACE_RANDOM_DESC"
 	}
+}
+static_portraits = {
```
Comment (MemeWizard) [inline common/game_rules/000_wc_game_rules.txt]: don't have localization

Comment (sejtalk) [reply common/game_rules/000_wc_game_rules.txt]: fixed

**Comment by geffe1** [comment]
ck2_381
She loses her new portraits as soon as she turns 40- on top of that.. the background on the new one is static.

>Added portraits that were supposed to be added in TFT branch but are added here because the branch is on hold.

Like Nathanos? Because.. he is sort of alive and thus won't have a new portrait.
ck2_376

**Comment by sejtalk** [comment]
> ck2_381
> She loses her new portraits as soon as she turns 40- on top of that.. the background on the new one is static.
> 
> > Added portraits that were supposed to be added in TFT branch but are added here because the branch is on hold.
> 
> Like Nathanos? Because.. he is sort of alive and thus won't have a new portrait.
> ck2_376

Backgrounds are always static for static portraits. Nathanos gets his portrait when he becomes undead.

**Pull request merged**